pom.xml

```diff
@@ -379,7 +379,7 @@
         <!-- Surefire versions are a minefield of bugs: be careful with upgrades -->
         <version.surefire.plugin>3.0.0-M7</version.surefire.plugin>
         <version.surefire.plugin.java-version.asm>9.4</version.surefire.plugin.java-version.asm>
-        <version.failsafe.plugin>${version.surefire.plugin}</version.failsafe.plugin>
+        <version.failsafe.plugin>3.0.0-M8</version.failsafe.plugin>
```

Comment:
Looks like dependabot made a mess here, would you mind fixing it?

Reply:
Perhaps it would be better to remove the property `version.failsafe.plugin` altogether and use `version.surefire.plugin` everywhere, so that dependabot doesn't make the same mistake next time.